lectures/networks.md

```diff
@@ -339,7 +339,7 @@ The Python package [Networkx](https://networkx.org/) provides a convenient
 data structure for representing directed graphs and implements many common
 routines for analyzing them.
 
-
+<!-- TODO -->
```

Comment:
What is this `<!--TODO-->` ?    (Inserted here and elsewhere.)

Reply:
This is an older version and all TODOs are removed in the latest file.